src/vaadin-menu-bar-sub-menu-mixin.html

```diff
@@ -138,6 +140,11 @@
     }
 
     _close() {
+      const listenOn = this._subMenu.listenOn;
```

Comment:
the reference to button which opened sub-menu is stored in `this._activeButton`
so you can use that one instead of checking for `localName`

Reply:
Thanks! Updated.